.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
How about using `4`? So far Dali has been developed with 4 spaces as indentation. Also with software projects, `4` seems to be the standard as normally too many indentations can indicate some code that can be refactored (e.g. many nested `if` blocks).

Reply:
ah - good catch!